### DISCUSSION TITLE
Update detection for LOAD IMAGE node

### DISCUSSION THREAD

**Discussion by DocChopper** [Comfy-Org/ComfyUI#6240] [Ideas]
As I am relying really heavily on ControlNet workflows to bring my drawings and sketches as templates into Comfy. I often tinker and finetune said templates if I see fit. But it can be time consuming, to alway load the images anew into your workflow. Especially when they are also used for masking.  

A feauture, that lets the LOAD IMAGE node detect if a loaded image has been changed. And maybe add some kind of "refresh source file" button or something. That be terrific. Also for any IMG2IMG workflow.

Thanks for considering.

**Comment by LukeG89** [comment]
You can press "R" to refresh

**Reply by DocChopper** [reply]
That may refresh the node. But it does not update any changes to a loaded picture.

**Reply by LukeG89** [reply]
The problem is the preview, because if you edit and save the image from an external software, you don't even need to hit refresh:

https://github.com/user-attachments/assets/0e04a4bf-06d3-4285-ab8e-f98ee5d098d5

But to be fair, I think that when you refresh, the previews should update as well.

**Reply by LukeG89** [reply]
I've opened a feature request in the frontend repo, let's see if it's possible to refresh the preview just by pressing "R":
- https://github.com/Comfy-Org/ComfyUI_frontend/issues/2082

**Comment by ltdrdata** [comment]
Since the issue has been moved to the front repo, I am closing this issue.
